src/scheduler/kubernetes/scheduler.py

```diff
@@ -817,28 +817,26 @@ def handle_timeouts(self):
 
     def upload_console(self, job_id):
         cursor = self.conn.cursor()
+        cursor.execute("begin")
```

Comment:
if we throw an exception after the tx has been started then next iteration of the handle loop will re-use the connection and also the transaction and probably end up in some weird state. In case of an exception the transaction has to be aborted.